quiz-application/server/src/main/java/com/quizapplication/server/service/impl/UserProfileServiceImpl.java

```diff
@@ -53,4 +56,55 @@ public ResponseEntity<Object> saveUserData(@Valid UserProfileDTO userProfileDTO,
 
         return new ResponseEntity<Object>(data, HttpStatus.OK);
     }
+
+    @Override
+    public ResponseEntity<Object> getUserData(RequestHeaderDTO requestHeaderDTO) {
+        String message = "";
+        LinkedHashMap<String, Object> data = null;
+
+        if (requestHeaderDTO.getPhone() == null) {
+            message = CodeConstant.BAD_REQUEST_MSG;
+
+            data = commonResponseEntity.ResponseEntityWithMap(CodeConstant.BAD_REQUEST_STATUS, message,
```

Comment:
write a separate method for this and pass the values
```suggestion
            data = response(CodeConstant.BAD_REQUEST_STATUS, message)
```